core/arch/arm/plat-sam/sama7g5.h

```diff
@@ -0,0 +1,400 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Header file for ATSAMA7G54
+ *
+ * Copyright (c) 2023 Microchip Technology Inc. and its subsidiaries.
+ *
+ * Subject to your compliance with these terms, you
```

Comment:
Do you need all this text? The reason I'm asking is that I suspect that it doesn't match the license defined by "BSD-2-Clause" exactly so we don't know what license this file has.